### ISSUE TITLE
Works in my windows 11 environment, but does not work in Centos 7 environment, reports unable to connect: speech.platform.bing.com:443 

### ISSUE THREAD

**Issue by 5UFKEFU** [rany2/edge-tts#4]
python3 --version
Python 3.9.10

same lan.
```
[root@sh1-mbp-122 bin]# ./edge-tts  --voice zh-CN-YunyangNeural  --text "朋友们或许还有一个疑问，就是不知道科大讯飞和neospeech哪个更好。据我的体会，它们其实各有优点。" > hello.mp3
Traceback (most recent call last):
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/site-packages/aiohttp/connector.py", line 986, in _wrap_create_connection
    return await self._loop.create_connection(*args, **kwargs)  # type: ignore[return-value]  # noqa
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/asyncio/base_events.py", line 1056, in create_connection
    raise exceptions[0]
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/asyncio/base_events.py", line 1041, in create_connection
    sock = await self._connect_sock(
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/asyncio/base_events.py", line 955, in _connect_sock
    await self.sock_connect(sock, address)
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/asyncio/selector_events.py", line 502, in sock_connect
    return await fut
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/asyncio/selector_events.py", line 507, in _sock_connect
    sock.connect(address)
OSError: [Errno 101] Network is unreachable

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/root/.cache/activestate/4fab4d29/usr/bin/./edge-tts", line 8, in <module>
    sys.exit(main())
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/site-packages/edge_tts/util.py", line 162, in main
    asyncio.run(_main())
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/site-packages/edge_tts/util.py", line 155, in _main
    await _tts(args)
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/site-packages/edge_tts/util.py", line 33, in _tts
    async for i in tts.run(
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/site-packages/edge_tts/communicate.py", line 259, in run
    async with session.ws_connect(
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/site-packages/aiohttp/client.py", line 1138, in __aenter__
    self._resp = await self._coro
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/site-packages/aiohttp/client.py", line 776, in _ws_connect
    resp = await self.request(
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/site-packages/aiohttp/client.py", line 535, in _request
    conn = await self._connector.connect(
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/site-packages/aiohttp/connector.py", line 542, in connect
    proto = await self._create_connection(req, traces, timeout)
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/site-packages/aiohttp/connector.py", line 907, in _create_connection
    _, proto = await self._create_direct_connection(req, traces, timeout)
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/site-packages/aiohttp/connector.py", line 1206, in _create_direct_connection
    raise last_exc
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/site-packages/aiohttp/connector.py", line 1175, in _create_direct_connection
    transp, proto = await self._wrap_create_connection(
  File "/root/.cache/activestate/4fab4d29/usr/lib/python3.9/site-packages/aiohttp/connector.py", line 992, in _wrap_create_connection
    raise client_error(req.connection_key, exc) from exc
aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host speech.platform.bing.com:443 ssl:default [Network is unreachable]
```

**Comment by rany2** [comment]
Does `curl -v https://speech.platform.bing.com` have any success?

**Comment by 5UFKEFU** [comment]
It was probably a problem with the ssl module in python 3.10, switching back to python 3.8 has solved it.
Thanks

**Comment by rany2** [comment]
Thanks, so this issue should be closed? FYI I'm on 3.10 and could use this module just fine.

**Comment by rany2** [comment]
I'll close this issue because I haven't received any response. Feel free to reopen if still relevant.
